src/containers/guest-home-page/how-it-works/HowItWorks.jsx

```diff
@@ -40,21 +40,21 @@ const HowItWorks = () => {
           {t('guestHomePage.howItWorks.title')}
         </Typography>
         <AppContentSwitcher
-          active={isStudent}
+          active={isTutor}
           onChange={onChange}
           styles={styles.switch}
           switchOptions={switchOptions}
           typographyVariant={'h6'}
         />
         <CardsWithButton
-          array={isStudent ? tutorCardBoxArray : studentCardBoxArray}
+          array={isTutor ? tutorCardBoxArray : studentCardBoxArray}
           btnText={
-            isStudent
+            !isTutor
               ? t('guestHomePage.whatCanYouDo.learn.actionLabel')
               : t('guestHomePage.whatCanYouDo.teach.actionLabel')
           }
-          isStudent={isStudent}
-          role={isStudent ? tutor : student}
+          isTutor={isTutor} //the name of this property is changed
```

Comment:
You have to change the property `isStudent` also inside the CardsWithButton component.
And we don't need this comment in the code.